.github/workflows/createTagFromDispatch.yml

```diff
@@ -0,0 +1,20 @@
+name: Create tag (notified from dispatch event)
```

Comment:
```suggestion
name: Create tag (notified from Radium-Release dispatch event)
```

Reply:
I didn't mention Radium-Release here, as we might have other trigger in the future (does not have to be from Radium-Release)